src/components/Molecules/CardDs/CardDs.js

```diff
@@ -139,7 +139,7 @@ const CardDs = ({
     return null;
   };
 
-  const external = target === 'blank' ? 'noopener noreferrer' : null;
+  const external = target === 'blank' ? 'noopener' : null;
```

Comment:
I take it `noreferrer` has been removed for SEO reasons?

Reply:
yeah you can you either not both.